src/ProjectNeon/Assets/Scripts/HPBarController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class HPBarController : MonoBehaviour
+{
+    #region Parametrs
```

Comment:
Don't use regions in this code base. Anytime we need regions, we should be refactoring smaller components.